task.py

```diff
@@ -300,23 +300,33 @@ def render_file(
     def initialize(self) -> None:
         pass
 
+    @property
+    def client(self) -> K8sClient:
+        return self.tc.client(tenant=self.tenant)
+
+    def _run_oc_namespace(
```

Comment:
We aren't really running anything. Could we give this a better name?
I.e. get_namespace_from_args